### DISCUSSION TITLE
Jumps to the end when player reaches an exact point in the video

### DISCUSSION THREAD

**Discussion by wbadlands** [mifi/lossless-cut#1443] [Support]
Hello. Like titles says, when I'm playing a video, it jumps to the end when it reaches certain points within the video. Whenever it gets to any of these points while playing (be it normal speed, or accelerated), it will jump to the end of the video.

If you manually click on the timeline to bypass the error point, it will continue playing as normal until reaching the next point that makes it jump to the end.

Console does give an error when this happens:

ffmpeg.js:50 ffprobe -v error -read_intervals '0%36.358' -show_packets -select_streams 0 -show_entries 'packet=pts_time,flags' -of json 'C:\LosslessCut\00001.mp4'

Video recorded with OBS, NVENC H264/AVC encoder, using CQP level 16.

Version 3.49.0

Tested in version 3.47.1, same problem.

I have a sample video that causes this to happen (6 seconds, 474 MB), can share if need be.

Thanks!

**Comment by mifi** [comment]
Hi. Is that ffprobe line the only error log youre getting?

**Reply by wbadlands** [reply]
Hey there. Yep, thats the only line.

Here's a small piece of video that causes this:

https://drive.google.com/u/0/uc?id=1GluFb1nAPogLcnXAEfR-pGBSGnMsJP37&export=download

Are you able to download it? The jump to the end happens around 4.2 seconds mark, gotta let the video play out tho. If you manually jump to a very close point, you might bypass the bug.

**Comment by wbadlands** [comment]
Tested recording using x264 (OBS) instead of h264 and I dont seem to be getting the issue on LosslessCut. So looks like the bug has to do with h264 and h265. (tested the latter and also had the "jump to the end" problem)

**Reply by mifi** [reply]
thanks for sharing the file. I can confirm the issue also. Actually I also tried to drag-drop the file into my Chrome browser (108.0.5359.124) and I got the same behavior there. So this is a bug in chromium (which losslesscut is also based on).

**Reply by wbadlands** [reply]
No worries, happy to help. Yes, i tried playing it in Chrome here and got the same result. But I guess, while its nice to have found where the bug lies, a fix for this is now out of your hands. I'll keep using x264 for now Thanks for getting in touch, love your app. Cheers!

**Reply by wbadlands** [reply]
It looks Chrome player has patched this bug. I've been editing H264 videos recorded with OBS with the latest version of LosslessCut and not getting this problem anymore.

**Reply by mifi** [reply]
ok cool, thanks fo reporting back!

**Reply by wbadlands** [reply]
Actually, this is still happening.  Video recorded using latest OBS, NVENC H264 encoder (tried a bunch of settings within this encoder with no success). Using latest Nvidia drivers aswell. Losslesscut 3.56.0.

In a 1 hour video, there are several points in which the video will jump to the end if I let it play. I get around it by using the arrow keys or mouse wheel to skip over these problematic points.

Not that you can do anything about it since problem seems to be with Chrome player, but just letting you know. Thanks!

**Reply by gbl08ma** [reply]
Also seeing the same thing with H264 videos encoded by OBS+NVENC. Losslesscut 3.59.1.

**Reply by 2canpop** [reply]
still happening in 3.64.0.0

**Reply by mifi** [reply]
With this file? https://drive.google.com/u/0/uc?id=1GluFb1nAPogLcnXAEfR-pGBSGnMsJP37&export=download

**Reply by KuePasta** [reply]
I am also still having this issue in 3.64.0.0, you can skip past it like the others have said, it's just more annoying than anything.
